src/image/ExposureFitsReader.cc

```diff
@@ -92,8 +243,17 @@ class ExposureFitsReader::MetadataReader {
             metadata = primaryMetadata;
         }
 
-        filter = Filter(metadata, true);
-        detail::stripFilterKeywords(metadata);
+        // Earlier versions persisted Filter as header keyword, version 2 persists FilterLabel as a Storable
+        if (version < 2) {
+            std::string key = "FILTER";
+            if (metadata->exists(key)) {
+                // Original Filter code depended on Boost for string trimming.
+                // DIY to avoid making this module depend on Boost.
```

Comment:
Thanks for the clarification (and also, there's not a string trim in the STL???).